cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -641,7 +641,7 @@ pub mod bridging {
 			/// (initially was calculated by test `OutboundQueue::calculate_fees` - ETH/ROC 1/400 and fee_per_gas 20 GWEI = 2200698000000 + *25%)
 			/// Needs to be more than fee calculated from DefaultFeeConfig FeeConfigRecord in snowbridge:parachain/pallets/outbound-queue/src/lib.rs
 			/// Polkadot uses 10 decimals, Kusama and Rococo 12 decimals.
-			pub const DefaultBridgeHubEthereumBaseFee: Balance = 2_750_872_500_000;
+			pub const DefaultBridgeHubEthereumBaseFee: Balance = 3_833_568_200_000;
```

Comment:
Could probably add the value here and then reuse: https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/constants/src/westend.rs#L172 Not a major issue though